src/community/json-ld/src/main/java/org/geoserver/jsonld/builders/AbstractJsonBuilder.java

```diff
@@ -12,17 +12,14 @@
 
     protected String key;
 
-    protected boolean isFeaturesField;
-
     public AbstractJsonBuilder(String key) {
-        this.isFeaturesField = key != null && key.equalsIgnoreCase("features");
         this.key = key;
     }
 
     public AbstractJsonBuilder() {}
 
     protected void writeKey(JsonLdGenerator writer) throws IOException {
-        if (key != null && !key.equals("") && !isFeaturesField) writer.writeFieldName(key);
+        if (key != null && !key.equals("")) writer.writeFieldName(key);
```

Comment:
I would suggest:
```
/**
     * Will write the key if it is neither empty or NULL.
     */
    protected void writeKey(JsonLdGenerator writer) throws IOException {
        if (key != null && !key.trim().isEmpty()) {
            writer.writeFieldName(key);
        } else {
            // should we throw an exception at least log something?
        }
    }
```

If the key is NULL or empty, is a normal situation? should we not throw an exception or at least log something?

Reply:
fixed